src/makeCalculator.js

```diff
@@ -37,7 +37,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
+
+    divide(number) {
+      this.result /= number;
+    },
```

Comment:
Methods must accept 2 numbers, the current value and the previous value and return just a value of operation. Pay attention to the use of the `operate` method in the examples

Reply:
Could you please explain why  I need 2 numbers for these methods? I have a "result" as the first operand and a "number" as the second operand.

Reply:
In the task methods "add", "subtract", "multiply", and "divide" do not have to "return" anything.